src/test/java/com/google/api/codegen/testdata/java_main_library.baseline

```diff
@@ -1820,6 +1810,106 @@ public class LibraryServiceApi implements AutoCloseable {
     return getBookFromArchiveCallable;
   }
 
+  // ----- updateBookIndex -----
+
+  // AUTO-GENERATED DOCUMENTATION AND METHOD - see instructions at the top of the file for editing.
+  /**
+   * Updates the index of a book.
+   *
+   * Sample code:
+   * <pre><code>
+   * try (LibraryServiceApi libraryServiceApi = LibraryServiceApi.createWithDefaults()) {
+   *   String formattedName = LibraryServiceApi.formatBookName("[SHELF]", "[BOOK]");
+   *   String indexName = "default index";
+   *   String indexMapItem = "";
+   *   Map&lt;String, UpdateBookIndexRequest.IndexMapEntry&gt; indexMap = new HashMap&lt;&gt;();
```

Comment:
`UpdateBookIndexRequest.IndexMapEntry` looks peculiar for the value type, is that right? (I haven't used map fields in proto-generated classes yet)